roles/wireguard/templates/wg0.conf.j2

```diff
@@ -0,0 +1,25 @@
+# Configuration managed by Ansible
+[Interface]
+Address = {{ wireguard_subnet }}
+ListenPort = {{ wireguard_port }}
+PrivateKey = {{ wg_priv_key['content'] | b64decode | trim }}
+
+PostUp = ip route add local {{ wireguard_subnet }} dev eth0
```

Comment:
Sure this is needed? Shouldn't `AllowedIPs` do this for us?

Reply:
Haven't looked into it. This was specifically added in a separate PR here https://github.com/python-discord/infra/pull/20 `AllowedIPs` existed prior to this PR too.